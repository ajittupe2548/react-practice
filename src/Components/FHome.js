import React, {useEffect} from 'react'

function FHome(props) {
    console.log(`*****Output is :  => FHome => FHome`,props.counter)
    useEffect(() => {
    console.log(`*****Output is :  => useEffect => useEffect`,)
    }, [])

    return (
        <div>
            <h1>Hi from functional home page functional {props.counter}</h1>
        </div>
    )
}

export default FHome;
