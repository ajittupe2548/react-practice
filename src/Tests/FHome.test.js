import renderer from 'react-test-renderer';
import FHome from '../Components/FHome';

it('render', () => {
    const component = renderer.create(
        <FHome/>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
