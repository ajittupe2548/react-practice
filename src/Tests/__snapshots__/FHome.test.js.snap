// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div>
  <h1>
    Hi from functional home page functional
  </h1>
</div>
`;
